install/ontology/oauth-consumer.rdf

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+<rdf:RDF
+        xml:base="http://www.taotesting.com/ontologies/taooauth.rdf#"
+        xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+        xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+>
+
+    <!-- Class to represent oauth2 consumer -->
+    <rdf:Description rdf:about="http://www.taotesting.com/ontologies/taooauth.rdf#Oauth-consumer">
```

Comment:
Just to clarify: if we have https://github.com/oat-sa/tao-core/blob/aeb1434d001dafd932eaef6da460fb43bd504237/models/ontology/oauth.rdf#L8 another oAuthConsumer and here we have oAuth2, maybe we could to split them, don't we? I mean that instead of Oauth-consumer we could use Oauth2-consumer to have defined type of oauth

Reply:
also taooath.rdf would be taooauth2.rdf